libs/ledger-live-common/src/families/solana/exchange.ts

```diff
@@ -0,0 +1,16 @@
+import { bip32asBuffer } from "@ledgerhq/hw-app-btc/bip32";
```

Comment:
Really not a good pattern here that `solana` has a dependency to the `hw-app-btc` lib, but I can see other families are already doing it so... not a blocker here.

Reply:
Yes I preferred to keep it consistent :angel: